src/manifest/dbtProject.ts

```diff
@@ -1113,6 +1117,30 @@ select * from renamed
     );
   }
 
+  checkAndShowDiagnosticsErrors() {
+    const allDiagnostics = [
+      this.rebuildManifestDiagnostics,
+      this.pythonBridgeDiagnostics,
+      this.projectConfigDiagnostics,
```

Comment:
Change order and show only first found error.

```suggestion
      this.projectConfigDiagnostics,
      this.pythonBridgeDiagnostics,
      this.rebuildManifestDiagnostics,
```